interface/patient_file/history/encounters.php

```diff
@@ -401,7 +422,7 @@ function changePageSize() {
 
             $countRes = sqlStatement($countQuery, $sqlBindArray);
             $count = sqlFetchArray($countRes);
-            $numRes = $count['c'];
+            $numRes += $count['c'];
```

Comment:
It looks like this didn't actually change the code logic, was thinking it did, but on further examination it keeps the code the same.

Reply:
oh yes, apologies: - i've been working on this code to sort out the problems where the documents are displayed on every page, issue #7275, and this new line has crept in from that work. It's not needed simply to add the print functionality. I'll fix and resubmit with a new PR